tests/e2e/tests/03-enable-serve-via-manual/02-manual-scale.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl scale deployment/target-deployment --replicas=1
+    namespaced: true
+  - command: sleep 2
+  - command: kubectl wait --for=condition=Ready pods -l app=target-deployment
+    namespaced: true
+  - command: sleep 2
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)